File: DSA basic q for beg/isperfectsqrornot.py
import math

def isperfectsqrornot(x):
    if(x>=0):
        sr = int(math.sqrt(x))
        if sr*sr == x:
            print("perfect square root")
        else:
            print("not a perfect square root")

x = 2500
isperfectsqrornot(x)

File: DSA basic q for beg/java dsa beginers/Isperfectsqrornot.java
public class Isperfectsqrornot {

    void isPerfectsqr(int x){
        int sr = (int) Math.sqrt(x);
        if (sr*sr == x){
            System.out.println("it is a perfect square");
        }
        else{
            System.out.println("not a perfect square");
        }
    

    }
    public static void main(String[] args) {
        Isperfectsqrornot is = new Isperfectsqrornot();
        int x = 2500;
        is.isPerfectsqr(x);
      
    }
    
}
